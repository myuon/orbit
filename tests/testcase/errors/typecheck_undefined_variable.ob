fun main() do
  let x = unknown_var;
end