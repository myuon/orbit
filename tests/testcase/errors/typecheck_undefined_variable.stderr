Undefined variable: unknown_var